include/swift/IDE/Utils.h

```diff
@@ -543,6 +543,19 @@ class SourceEditOutputConsumer : public SourceEditConsumer {
   void accept(SourceManager &SM, RegionType RegionType, ArrayRef<Replacement> Replacements) override;
 };
 
+class DuplicatingSourceEditConsumer : public SourceEditConsumer {
+  SourceEditTextConsumer TextConsumer;
+  SourceEditOutputConsumer OutputConsumer;
+
+public:
+  DuplicatingSourceEditConsumer(llvm::raw_ostream &TextStream,
+                                SourceManager &SM, unsigned BufferId,
+                                llvm::raw_ostream &OutputStream);
+  ~DuplicatingSourceEditConsumer();
```

Comment:
Is it necessary to specify the destructor for the code to work? If not, I would remove it since it’s not doing anything.